about.html

```diff
@@ -0,0 +1,93 @@
+            <a href="#">Program</a>
+            <a href="#">Join</a>
+            <a href="#">Sponsor</a>
+            <a href="#">News</a>
+            <div class="campaign">
+                <a href="#">CC Campaign</a>
+            </div>
+        </nav>
+    </header>
+    <section class="home-pages">
+        <div class="home">
+            <h1>CREATIVE COMMONS</br>GLOBAL SUMMIT 2015</h1>
+        </div>
+        <div class="paragraphab">
+            <p>This is the Generic Global Summit, where we bring together the least influential and unimportant leaders from around the block to flap their gums about the non-issues and gossips of our time.A joyful celebration bellieving in the value of opennes and sharing, creating This is the Generic Global Summit, where we bring together the least influential and unimportant leaders from around the block to flap their gums about the non-issues and gossips of our time.A joyful celebration bellieving in the value<br> A posative change with people from over 80 countries is taking place in October, in Korea A posative change with people from over 80 countries is taking place in October, in Korea</p>
+        </div>
+        <div class="hom-foots">
+            <p>Please avoid doing anything. Here you can ask nothing</p>
+            <h4>rofl@gcgn.com</h4>
+        </div>
+    </section>
+    <section class="m-page">
+        <div class="primary">
+            <p>Creative Commons Global Summit 2015 Logo</p>
+            <div class="hlsa"></div>
+        </div>
+        <div class="secondary">
+            <p>This is the Generic Global Summit, where we bring together where we bring together where we bring the least influential and unimpThis is the Generic Global Summit, where we bring together the least influential and unimpThis is the Generic Global Summit, where we bring together the least influential and unimp</p>
+        </div>
+        <div class="third">
+            <img src="images/logo.png">
+        </div>
+    </section>
+    <hr class="just">
+    <section class="m-page">
+        <div class="primary">
+            <p>See the past CC Global Summits</p>
+            <div class="hlsa"></div>
+        </div>
+        <div class="secondary">
+            <p>This is the Generic Global Summit, where we bring together where we bring together where we bring the</p>
+        </div>
+        <div class="thirds">
+            <div class="thirdl">
+                <h1>2015</h1>
+                <p>CC Global Summits 2015 in Ambo</p>
+            </div>
+            <div class="thirdl">
+                <h1>2015</h1>
+                <p>CC Global Summits 2015 in Ambo</p>
+            </div>
+        </div>
+    </section>
+    <section class="footers">
+        <div class="foot-logo">
+            <img src="images/footer-logo-dark.svg">
+        </div>
+        <div class="copys">
+            <h4>Copyright 2017</h4>
+            <p>18,Valluvar street,Kodambakkam,Chennai</p>
+        </div>
```

Comment:
- [ ] [REQUIRED] Good job so far! Please add correct `Layout` to `footer` part for mobile like [Behance Design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) and pay attention to sizes of your cards🙏🏼 This review for `[MOBILE]` version!

![image](https://user-images.githubusercontent.com/63915024/225588466-907cfd04-ab66-4507-8012-1ded726b8b3f.png)